test/TestBuilder.cpp

```diff
@@ -229,11 +241,35 @@ TEST_CASE("JsonBuilder string push_back")
 TEST_CASE("JsonBuilder chrono push_back", "[builder]")
 {
     auto now = std::chrono::system_clock::now();
+    auto nowTicks = std::chrono::time_point_cast<ticks>(now);
```

Comment:
Note: system_clock values no longer guaranteed to perfectly round-trip through JsonBuilder -- they will now be rounded to the nearest 100ns unit.